doc/commands.doc

```diff
@@ -714,7 +714,7 @@ Structural indicators
   \ref cfg_force_local_includes "FORCE_LOCAL_INCLUDES" to \c YES.
 
   To disable the include information altogether set
-  \ref cfg_show_include_files "SHOW_INCLUDE_FILES" to \c NO.
+  \ref cfg_show_header "SHOW_HEADERFILE" to \c NO.
```

Comment:
GitHub Actions failed with:
/home/runner/work/doxygen/doxygen/build/doc/commands.doc:717: warning: unable to resolve reference to 'cfg_show_header' for \ref command

Reply:
Yes, I renamed the option and missed updating that one.

```suggestion
  \ref cfg_show_headerfile "SHOW_HEADERFILE" to \c NO.
```

Reply:
Best it to update the pull request with this.

Reply:
Yes, done. I tried to run the `make docs` target locally to check it was fixed, but it doesn't work and I don't know why, the error isn't very helpful:

```
[100%] Built target run_doxygen
[100%] Generating Doxygen Manual PDF.
This is pdfTeX, Version 3.14159265-2.6-1.40.21 (TeX Live 2020) (preloaded format=pdflatex)
 \write18 enabled.
entering extended mode
(./doxygen_manual.tex
LaTeX2e <2020-02-02> patch level 5
L3 programming layer <2020-04-06>
make[3]: *** [doc/CMakeFiles/doxygen_pdf.dir/build.make:81: doc/CMakeFiles/doxygen_pdf] Error 1
make[2]: *** [CMakeFiles/Makefile2:720: doc/CMakeFiles/doxygen_pdf.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:700: doc/CMakeFiles/docs.dir/rule] Error 2
make: *** [Makefile:386: docs] Error 2
```

So I've pushed and I'll just wait for the Actions results.

Reply:
Due to a regression (see also #8653) the output master is currently, sometimes, hard to read .
Most likely the problem is a bit earlier in the output that you see on the screen.

Reply:
Unless it's very well hidden, there is no other error shown.

```
$ make docs          
[  4%] Built target doxycfg
[  4%] Built target md5
[  5%] Built target lodepng
[ 11%] Built target mscgen
[ 11%] Checking the git repository for changes...
[ 11%] Built target check_git_repository
[ 11%] Checking the doxygen version for changes...
[ 11%] Built target check_doxygen_version
[ 12%] Built target doxygen_version
[ 14%] Built target xml
[ 14%] Built target generate_configvalues_header
[ 16%] Built target vhdlparser
[ 55%] Built target doxymain
[ 55%] Built target doxygen
[ 79%] Built target examples
[ 80%] Generating Latex and HTML documentation.
Doxygen version used: 1.9.2 (39f2a183e27a5d5c43dcd49dc7999b55cbb67450)
Searching for include files...
Searching for example files...
Searching for files in directory /home/jwakely/src/doxygen/build/examples
Searching for images...
Searching for files in directory /home/jwakely/src/doxygen/build/doc
Searching for files in directory /home/jwakely/src/doxygen/build/doc/images
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for files to exclude
Searching INPUT for files to process...
Reading formula repository...
Reading and parsing tag files
Parsing files
Reading /home/jwakely/src/doxygen/build/doc/index.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/index.doc...
Reading /home/jwakely/src/doxygen/build/doc/install.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/install.doc...
Reading /home/jwakely/src/doxygen/build/doc/starting.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/starting.doc...
Reading /home/jwakely/src/doxygen/build/doc/docblocks.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/docblocks.doc...
Reading /home/jwakely/src/doxygen/build/doc/markdown.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/markdown.doc...
Reading /home/jwakely/src/doxygen/build/doc/lists.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/lists.doc...
Reading /home/jwakely/src/doxygen/build/doc/grouping.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/grouping.doc...
Reading /home/jwakely/src/doxygen/build/doc/formulas.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/formulas.doc...
Reading /home/jwakely/src/doxygen/build/doc/tables.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/tables.doc...
Reading /home/jwakely/src/doxygen/build/doc/diagrams.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/diagrams.doc...
Reading /home/jwakely/src/doxygen/build/doc/preprocessing.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/preprocessing.doc...
Reading /home/jwakely/src/doxygen/build/doc/autolink.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/autolink.doc...
Reading /home/jwakely/src/doxygen/build/doc/output.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/output.doc...
Reading /home/jwakely/src/doxygen/build/doc/searching.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/searching.doc...
Reading /home/jwakely/src/doxygen/build/doc/extsearch.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/extsearch.doc...
Reading /home/jwakely/src/doxygen/build/doc/customize.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/customize.doc...
Reading /home/jwakely/src/doxygen/build/doc/custcmd.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/custcmd.doc...
Reading /home/jwakely/src/doxygen/build/doc/external.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/external.doc...
Reading /home/jwakely/src/doxygen/build/doc/faq.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/faq.doc...
Reading /home/jwakely/src/doxygen/build/doc/trouble.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/trouble.doc...
Reading /home/jwakely/src/doxygen/build/doc/features.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/features.doc...
Reading /home/jwakely/src/doxygen/build/doc/doxygen_usage.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/doxygen_usage.doc...
Reading /home/jwakely/src/doxygen/build/doc/doxywizard_usage.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/doxywizard_usage.doc...
Reading /home/jwakely/src/doxygen/build/doc/config.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/config.doc...
Reading /home/jwakely/src/doxygen/build/doc/commands.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/commands.doc...
Reading /home/jwakely/src/doxygen/build/doc/htmlcmds.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/htmlcmds.doc...
Reading /home/jwakely/src/doxygen/build/doc/xmlcmds.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/xmlcmds.doc...
Reading /home/jwakely/src/doxygen/build/doc/emojisup.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/emojisup.doc...
Reading /home/jwakely/src/doxygen/build/doc/language.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/language.doc...
Reading /home/jwakely/src/doxygen/build/doc/perlmod.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/perlmod.doc...
Reading /home/jwakely/src/doxygen/build/doc/perlmod_tree.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/perlmod_tree.doc...
Reading /home/jwakely/src/doxygen/build/doc/arch.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/arch.doc...
Reading /home/jwakely/src/doxygen/build/doc/changelog.doc...
Parsing file /home/jwakely/src/doxygen/build/doc/changelog.doc...
Building macro definition list...
Building group list...
Building directory list...
Building namespace list...
Building file list...
Building class list...
Building concept list...
Computing nesting relations for classes...
Associating documentation with classes...
Associating documentation with concepts...
Building example list...
Searching for enumerations...
Searching for documented typedefs...
Searching for members imported via using declarations...
Searching for included using directives...
Searching for documented variables...
Building interface member list...
Building member list...
Searching for friends...
Searching for documented defines...
Computing class inheritance relations...
Computing class usage relations...
Flushing cached template relations that have become invalid...
Computing class relations...
Add enum values to enums...
Searching for member function documentation...
Creating members for template instances...
Building page list...
Search for main page...
Computing page relations...
Determining the scope of groups...
Sorting lists...
Determining which enums are documented
Computing member relations...
Building full member lists recursively...
Adding members to member groups.
Computing member references...
Inheriting documentation...
Generating disk names...
Adding source references...
Adding xrefitems...
Sorting member lists...
Setting anonymous enum type...
Computing dependencies between directories...
Generating citations page...
Counting members...
Counting data structures...
Resolving user defined references...
Finding anchors and sections in the documentation...
Transferring function references...
Combining using relations...
Adding members to index pages...
Correcting members for VHDL...
Computing tooltip texts...
Generating style sheet...
Generating search indices...
Generating images for formulas in HTML...
Generating example documentation...
Generating file sources...
Generating file documentation...
Generating page documentation...
Generating docs for page install...
Generating docs for page starting...
Generating docs for page docblocks...
Generating docs for page markdown...
Generating docs for page lists...
Generating docs for page grouping...
Generating docs for page formulas...
Generating docs for page tables...
Generating docs for page diagrams...
Generating docs for page preprocessing...
Generating docs for page autolink...
Generating docs for page output...
Generating docs for page searching...
Generating docs for page extsearch...
Generating docs for page customize...
Generating docs for page custcmd...
Generating docs for page external...
Generating docs for page faq...
Generating docs for page trouble...
Generating docs for page features...
Generating docs for page doxygen_usage...
Generating docs for page doxywizard_usage...
Generating docs for page config...
Generating docs for page commands...
Generating docs for page htmlcmds...
Generating docs for page xmlcmds...
Generating docs for page emojisup...
Generating docs for page langhowto...
Generating docs for page perlmod...
Generating docs for page perlmod_tree...
Generating docs for page arch...
Generating docs for page changelog...
Generating group documentation...
Generating class documentation...
Generating concept documentation...
Generating namespace index...
Generating graph info page...
Generating directory documentation...
Generating index page...
Generating page index...
Generating module index...
Generating namespace index...
Generating namespace member index...
Generating concept index...
Generating annotated compound index...
Generating alphabetical compound index...
Generating hierarchical class index...
Generating member index...
Generating file index...
Generating file member index...
Generating example index...
finalizing index lists...
writing tag file...
Running plantuml with JAVA...
lookup cache used 0/65536 hits=0 misses=0
finished...
[100%] Built target run_doxygen
[100%] Generating Doxygen Manual PDF.
This is pdfTeX, Version 3.14159265-2.6-1.40.21 (TeX Live 2020) (preloaded format=pdflatex)
 \write18 enabled.
entering extended mode
(./doxygen_manual.tex
LaTeX2e <2020-02-02> patch level 5
L3 programming layer <2020-04-06>
make[3]: *** [doc/CMakeFiles/doxygen_pdf.dir/build.make:81: doc/CMakeFiles/doxygen_pdf] Error 1
make[2]: *** [CMakeFiles/Makefile2:720: doc/CMakeFiles/doxygen_pdf.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:700: doc/CMakeFiles/docs.dir/rule] Error 2
make: *** [Makefile:386: docs] Error 2
```

Reply:
Only things I can think of are:
- open pdf file
- the log file on the latex directory might give some information (though will also be hard to read).

Reply:
I'm missing a LaTeX package that it uses.